src/core/drive/morphing_page_renderer.js

```diff
@@ -7,14 +6,16 @@ export class MorphingPageRenderer extends PageRenderer {
   static renderElement(currentElement, newElement) {
     morphElements(currentElement, newElement, {
       callbacks: {
-        beforeNodeMorphed: element => !canRefreshFrame(element)
+        beforeNodeMorphed: (node, newNode) => {
+          if (super.shouldRefreshChildFrameWithMorphing(null, node, newNode)) {
```

Comment:
I don't love passing `null` here. At first glance, it's surprising and it makes you wonder what this logic is about.

What do you think of splitting the logic in two functions: Here it could be `shouldReloadFrameWithMorphing(...) && !closestFrameReloadableWithMorphing(...)` and, in the frame renderer, it could be: `shouldReloadFrameWithMorphing(...) && !closestFrameReloadableWithMorphing(...) == currentElement`